src/NHibernate/Util/StringHelper.cs

```diff
@@ -872,7 +872,7 @@ public static bool IsAnyNewLine(this string str, int index, out int newLineLengt
 			return false;
 		}
 
-		public static string GenerateSuffix(int index)
+		internal static string GenerateSuffix(int index)
```

Comment:
Introduced in #3640, not yet released, so, ok.

Reply:
Thanks. I've actually done the change in the PR, but forgot to push :-/